R/msaGaugeLinearity.R

```diff
@@ -86,7 +86,7 @@ msaGaugeLinearity <- function(jaspResults, dataset, options, ...) {
   if (ready) {
 
     if (length(dataset[[measurements]]) < 2){
-      table2$setError(gettextf("T-Test requires more than 1 measurement. %i valid measurement(s) detected in %s.", length(dataset[[measurements]]), measurements))
+      table2$setError(gettextf("T-Test requires more than 1 measurement. $1%i valid measurement(s) detected in $2%s.", length(dataset[[measurements]]), measurements))
```

Comment:
Hi @boutinb , This seems to be problematic, it should be `%1$i` and `%2$s` otherwise `$ `will be printed before each placeholder, could you review this PR https://github.com/jasp-stats/jaspQualityControl/pull/256 for these changes.